app/controllers/distributions_controller.rb

```diff
@@ -89,7 +89,7 @@ def show
   def edit
     @distribution = Distribution.includes(:line_items).includes(:storage_location).find(params[:id])
     if (!@distribution.complete? && @distribution.future?) || current_user.organization_admin?
-      @distribution.line_items.build
+      @distribution.line_items.build if @distribution.line_items.size.zero?
```

Comment:
Hmm, I wonder if this behavior makes sense. I could see a user commonly adding new line items when editing an existing distribution, so they might want to see a placeholder as soon as they click edit. What do you think @fosterv2 @edwinthinks @designbygia ?

Reply:
We talked about it in Zoom with @seanmarcia . Think it makes sense to go with Valerie's change, since users are usually not adding a large quantity of items manually _and_ since the `Add another item` button is right below the empty entry.